exec/memory/base/src/main/java/org/apache/drill/exec/memory/BaseAllocator.java

```diff
@@ -788,8 +788,7 @@ public static StringBuilder indent(StringBuilder sb, int indent) {
   public static enum Verbosity {
     BASIC(false, false), // only include basic information
     LOG(true, false), // include basic
-    LOG_WITH_STACKTRACE(true, true) //
-    ;
+    LOG_WITH_STACKTRACE(true, true); //
```

Comment:
Please remove the empty comment at the end of the line.

Reply:
done